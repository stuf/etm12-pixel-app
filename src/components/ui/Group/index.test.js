import * as React from 'karet';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import Index from './index';

it('matches snapshot', () => {
  const wrap = mount(<Index title="Group title">Group content</Index>);
  expect(toJson(wrap)).toMatchSnapshot();
});
