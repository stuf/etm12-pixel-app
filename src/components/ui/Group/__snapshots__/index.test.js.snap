// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Group
  title="Group title"
>
  <section
    className="root"
  >
    <header
      className="header"
    >
      Group title
    </header>
    <div
      className="content"
    >
      Group content
    </div>
  </section>
</Group>
`;
